web/libs/editor/src/components/SidePanels/OutlinerPanel/ViewControls.tsx

```diff
@@ -275,22 +272,22 @@ const ToggleRegionsVisibilityButton = observer<FC<ToggleRegionsVisibilityButton>
   const isAllHidden = !isDisabled && regions.isAllHidden;
 
   return (
-    <Elem
+    <Button
```

Comment:
since we switched to Button from Elem we no longer the `tag` to be passed in here